scripts/publish-functions.sh

```diff
@@ -63,12 +63,13 @@ for f in *; do
         queuename=${f//_/-}
         echo "About to publish function $functionname"
 
+        # shellcheck disable=SC2140
         gcloud functions deploy "$functionname" --trigger-http \
             --runtime nodejs10 \
             --region "$FUNCTION_REGION" \
             --set-env-vars DRIFT_PRO_BUCKET="$BUCKET",KEY_LOCATION="$KEY_LOCATION",KEY_RING="$KEY_RING",GCF_SHORT_FUNCTION_NAME="$functionname",PROJECT_ID="$PROJECT_ID",GCF_LOCATION="$FUNCTION_REGION",PUPPETEER_SKIP_CHROMIUM_DOWNLOAD="1"
```

Comment:
Does this work if it's all inside a single pair of quotes? I assume not because of the disabled check?

Reply:
Inside `'` the expressions don't expand (e.g. `$BUCKET`). It looks like it can be fixed if we make the `1` in single quotes like:

`PUPPETEER_SKIP_CHROMIUM_DOWNLOAD='1'`